EncounterHandler.js

```diff
@@ -690,16 +690,23 @@ function inject_monster_image(stat) {
 	}
 	if (window.EncounterHandler.combat_body.find(".encounter-details-content-section__content .injected-image").length == 0) {
 		let content = window.EncounterHandler.combat_body.find(".encounter-details-content-section__content");
-		let image = `<img style="width:100%" class="injected-image" src="${stat.data.largeAvatarUrl}"
-			alt="${stat.data.name}" class="monster-image" onerror="this.src='${stat.data.avatarUrl}'";onerror=''></img>`;
+		const image = `<img style="width:100%" class="injected-image" src="${stat.data.largeAvatarUrl}"
+			alt="${stat.data.name}" class="monster-image"></img>`;
+		$(image).on("error", () => {
```

Comment:
If you want, you can update this to be `(event) => {` which would allow you to get the image with `$(event.target)` rather than searching the entire iframe document for it. Not a big deal, but slightly more efficient.